### PR TITLE
PUBSTWO-1736 - initial commit

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: src/test/java/gov/usgs/cida/pubs/transform/CrossrefTransformerIT.java [hunk 1]
```diff
@@ -212,11 +190,12 @@ public void testPubWithoutSeriesNumber() throws IOException, UnsupportedEncoding
 
 		instance.writeResult(pub);
 		instance.end();
-		String output = new String(target.toByteArray(), PubsConstantsHelper.DEFAULT_ENCODING);
 
-		assertNotNull(output);
-		assertTrue(0 < output.length());
-		assertWellFormed(output);
+		String output = new String(target.toByteArray(), PubsConstantsHelper.DEFAULT_ENCODING);
+		assertTrue(StringUtils.isNotBlank(output), "The XML was null or empty.");
```
Comment (ssoper-usgs) [inline src/test/java/gov/usgs/cida/pubs/transform/CrossrefTransformerIT.java:195]: `isNotBlank` is pretty nifty.

**Review by ssoper-usgs — APPROVED** [review]

File: src/test/java/gov/usgs/cida/pubs/busservice/CrossRefBusServiceTest.java [hunk 2]
```diff
@@ -224,8 +227,7 @@ public void tetBuildCrossRefPost() throws IOException {
 		assertNotNull(content);
 		IOUtils.copy(content, baos);
 		String requestBody = baos.toString(PubsConstantsHelper.DEFAULT_ENCODING);
-		assertNotNull(requestBody);
-		assertTrue(0 < requestBody.length());
+		StringUtils.isNotBlank(requestBody);
```
Comment (mbucknell) [inline src/test/java/gov/usgs/cida/pubs/busservice/CrossRefBusServiceTest.java]: Do you need this in an "assertTrue"?

**Review by mbucknell — APPROVED** [review]
One little comment

File: src/test/java/gov/usgs/cida/pubs/busservice/CrossRefBusServiceTest.java (hunk 2 above)
Comment (dsteinich) [reply src/test/java/gov/usgs/cida/pubs/busservice/CrossRefBusServiceTest.java]: yes, thanks

**Pull request merged**